### ISSUE TITLE
[BUG] Vibration on Right Stick touch

### ISSUE THREAD

**Issue by inCrooo** [mKenfenheuer/steam-deck-windows-usermode-driver#82]
**Describe the bug**
-There's a light vibration when I touch the right stick. 
-It doesn't happen on the left stick, only the right.
-"Right stick touch" is not mapped to anything.
-The vibration on right stick touch persists even after disabling haptic feedback.

**Expected behavior**
No vibration on stick touch.

**Debug information**
Driver version: v0.3.2

**Additional context**
[SWICD_Support_638035180799042692.zip](https://github.com/mKenfenheuer/steam-deck-windows-usermode-driver/files/9962677/SWICD_Support_638035180799042692.zip)

**Comment by redphx** [comment]
I also have this problem on 0.3.2 after installing tetherscript. It happens on both sticks for me though. I tried to use the default `app_config.json` file but it doesn't fix the issue.

**Comment by inCrooo** [comment]
> I also have this problem on 0.3.2 after installing tetherscript. It happens on both sticks for me though. I tried to use the default `app_config.json` file but it doesn't fix the issue. I'm using Win 11 22H2.

You're right. It's both sticks. But for some reason the right stick vibration is much more noticeable.

I do have tetherscript installed. I uninstalled all tetherscript drivers one by one but the vibration still persists. I also tried default 
app_config.json and it didn't help.

The only thing that stops the vibration is pausing/closing SWICD.

I'm on Windows 10 21H2.

**Comment by inCrooo** [comment]
Sorry didn't mean to close it.

**Comment by inCrooo** [comment]
Sorry didn't mean to close it.

**Comment by redphx** [comment]
Did a clean Win 11 install and still have this bug.
